derp/derp_server.go

```diff
@@ -84,11 +84,20 @@ func init() {
 }
 
 const (
-	perClientSendQueueDepth = 32 // packets buffered for sending
-	writeTimeout            = 2 * time.Second
-	privilegedWriteTimeout  = 30 * time.Second // for clients with the mesh key
+	defaultPerClientSendQueueDepth = 32 // default packets buffered for sending
+	writeTimeout                   = 2 * time.Second
+	privilegedWriteTimeout         = 30 * time.Second // for clients with the mesh key
 )
 
+func getPerClientSendQueueDepth() int {
+	perClientSendQueueDepth, ok := envknob.LookupInt("DERP_PER_CLIENT_SEND_QUEUE_DEPTH")
```

Comment:
these normally start with `TS_` and in particular this one's probably `TS_DEBUG_`